!SystemError: with a message
	in System_error4.foo3() [System_error4.v3 @ 9:29]
	in System_error4.foo2() [System_error4.v3 @ 7:26]
	in System_error4.foo1() [System_error4.v3 @ 6:26]
	in System_error4.main() [System_error4.v3 @ 3:21]

